forms/UploadField.php

```diff
@@ -1498,6 +1498,9 @@ class UploadField_SelectHandler extends RequestHandler {
 	 */
 	protected $folderName;
 
+	// Set pagnation quantity for file list field 
```

Comment:
S’okay - Dan did it in the merge commit https://github.com/silverstripe/silverstripe-framework/commit/31a4e184f737ad4f23d3c15b80199198c60a17f0